apps-rendering/src/item.ts

```diff
@@ -307,6 +309,7 @@ const itemFields = (
 		),
 		logo: paidContentLogo(content.tags),
 		webUrl: content.webUrl,
+		promotedNewsletter: request.promotedNewsletter,
```

Comment:
This will automatically convert to an `Option`.

```suggestion
		promotedNewsletter: fromNullable(request.promotedNewsletter),
```